.travis.yml

```diff
@@ -1,11 +1,14 @@
-# see https://github.com/SebastianJay/unity-ci-test for how to set up unity builds using travis-ci
-
-# do not install anything extra onto the image
-language: generic
-
-# use a Mac OSX image
-os: osx
-
+sudo: required
+language: bash
+env:
+  global:
+  - IMAGE_NAME=gableroux/unity3d:2018.2.6f1
```

Comment:
Contrary to what https://github.com/UltraStar-Deluxe/Play/wiki/Compiling-the-game and #28 say, this uses a more recent Unity version. Wiki edits are probably separate from the codebase, and I'm guessing the newer version is deliberate here, but don't forget to update the PR and wiki after merging.